### ISSUE TITLE
Force display the notification when app is in foreground

### ISSUE THREAD

**Issue by ickata** [EddyVerbruggen/nativescript-local-notifications#50]
Hello,

Is there a chance to force show a local notification even though the app is being in foreground? Right now notification is not shown and instead `onMessageReceived` callback is invoked immediately (iOS).

Thanks!

**Comment by EddyVerbruggen** [comment]
Hi, have you ever seen a real notification while the app was in the foreground in any app? I don't think Apple/Google allows that..

**Comment by ickata** [comment]
Hi @EddyVerbruggen ,

I thought I've seen before posting this, and indeed Messenger (iOS) displays a "notification" when you are viewing a conversation and someone else sends you a message at the same time. Previously I thought that this is a local notification. But today I took a closer look, and it appears that this is some custom implementation which looks very close to local notification (but it isn't). 

When I searched more deep it looks like [showing local notification is not possible in iOS when app is running in foreground](https://stackoverflow.com/questions/9555564/is-it-possible-to-pop-up-local-notifications-while-the-iphone-app-is-on-screen). But it is possible in Android – I tested it with NativeScript app and your plugin – works for me (notification is shown in the tray).
